BossMod/Autorotation/RotationModule.cs

```diff
@@ -93,12 +93,9 @@ public abstract class RotationModule(RotationModuleManager manager, Actor player
 
     public virtual string DescribeState() => "";
 
-    // utility to check action/trait unlocks
-    public bool ActionUnlocked(ActionID action)
-    {
-        var def = ActionDefinitions.Instance[action];
-        return def != null && def.AllowedClasses[(int)Player.Class] && Player.Level >= def.MinLevel && (ActionDefinitions.Instance.UnlockCheck?.Invoke(def.UnlockLink) ?? true);
-    }
+    public virtual AI.Targeting? SelectTargetForAI(StrategyValues strategy, AI.Targeting aiDecision) => null;
```

Comment:
Why is this needed?

Reply:
just for range to target. EW bossmod had similar functionality. (specifically, red mage wants to be around 25y from target in most cases, and in melee range if they have enough gauge.)